core/src/Revolution/Processors/System/Dashboard/Widget/Feed.php

```diff
@@ -56,7 +56,12 @@ public function loadFeed($url)
     {
         $feed = new \SimplePie();
 
-        $feed->set_cache_location($this->modx->getOption('core_path') . 'cache/rss/');
+        $cachePath = $this->modx->getOption('core_path') . 'cache/rss/';
```

Comment:
I believe this needs to use the `cache_path` (or `xPDO::OPT_CACHE_PATH`) setting.